frontend/src/pages/Layout.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 import PATHS from '../constants/routes';
-import Header from "../components/layout/Header"
-import Menu from "../components/layout/Menu"
+import HeaderAuthorized from "../components/Layout/HeaderAuthorized"
+import Menu from "../components/Layout/Menu"
 
 const Layout: React.FC = () => (
   <BrowserRouter basename={PATHS.home}>
-    <Header />
+    <HeaderAuthorized />
```

Comment:
You can change the name back. For unauthorized clients we will use another type of layout